pkg/arvo/app/spider.hoon

```diff
@@ -554,9 +554,12 @@
     ~
   %+  give-simple-payload:app:server  u.eyre-id
   ^-  simple-payload:http
-  :_  ~  :_  ~
   ?.  ?=(http-error:spider term)
-    ((slog tang) 500)
+    %-  (slog tang)
+    =/  tube  (convert-tube %tang %json desk bowl)
+    %-  json-response:gen:server
```

Comment:
Shouldn't the HTTP status code here still be 500?  We did error out, after all.

Reply:
Yeah this is a good point. Is [this](https://github.com/urbit/urbit/pull/6636/commits/1818790e357c6c92f6d162b649f571ec073c62ff) an appropriate fix?